moto/glue/models.py

```diff
@@ -183,6 +183,32 @@ def create_job(
         )
         return name
 
+    def list_jobs(self, next_token, max_results, tags):
+        next_token = int(next_token) if next_token else 0
```

Comment:
What is the value/format that AWS uses for `next_token`?
If AWS uses a randomly generated string, it would be easier to use the `paginator`-decorator for these two methods.
That way we don't have to deal with the pagination logic ourselves.

See this bit in the docs with some guidelines on how to implement that: http://docs.getmoto.org/en/latest/docs/contributing/development_tips/utilities.html#paginator

(If AWS does use a simple number for the `next_token`, and we want to replicate that exact format, we can keep this implementation as is.)

Reply:
Changed accordingly 😄